src/swim/SConscript

```diff
@@ -27,13 +27,13 @@ SRC = ['swim.c']
 def scons():
     """Scons function"""
 
-    Import('env', 'prereqs', 'gurt_lib')
+    Import('env', 'gurt_lib')
 
     env.AppendUnique(LIBPATH=[Dir('.')])
 
     denv = env.Clone()
 
-    denv.AppendUnique(CPPPATH=['#/src/cart/src/swim'])
+    denv.AppendUnique(CPPPATH=['#/src/swim'])
```

Comment:
From the comment in the JIRA:
Moved the "src/cart/src/swim" directory to "src/swim"

While the original requirement is:
move src/cart/src/swim to src/cart/swim

There probably a reason for this change, can you please add a note to explain the change, similar to the other implementation vs requirement location differences explained?

Reply:
No, this was an error from my side. Good catch. I'm working on the fix. Thank you @GitHuaKuang